homeassistant/components/deconz/__init__.py

```diff
@@ -166,6 +174,8 @@ def async_add_remote(sensors):
                 _LOGGER.error('Could not find the entity %s', entity_id)
                 return
 
+            field = field + subpath
```

Comment:
subpath only exists if entity_id exist.

Reply:
Ooops. I didn't spend too much effort on correctness TBH, since I wanted to present the idea as a PoC first. Will fix it.